<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minecraft Universal In-Game Timer | Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="24" column="9" alignment="Qt::AlignLeft">
     <widget class="QRadioButton" name="light_theme_button">
      <property name="text">
       <string>Light</string>
      </property>
     </widget>
    </item>
    <item row="6" column="11">
     <widget class="QPushButton" name="browse_button">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="igt_timer_check">
      <property name="text">
       <string>IGT Timer</string>
      </property>
     </widget>
    </item>
    <item row="22" column="1" colspan="3" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLabel" name="opacity_min_text">
      <property name="text">
       <string>10%</string>
      </property>
     </widget>
    </item>
    <item row="22" column="11" alignment="Qt::AlignLeft|Qt::AlignVCenter">
     <widget class="QLabel" name="opacity_max_text">
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="theme_text">
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item row="22" column="4" colspan="7">
     <widget class="QSlider" name="opacity_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="10" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="hours_check">
      <property name="text">
       <string>Show Hours</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5" colspan="5" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="world_name_check">
      <property name="text">
       <string>World Name/Version</string>
      </property>
     </widget>
    </item>
    <item row="14" column="8" colspan="3" alignment="Qt::AlignLeft|Qt::AlignVCenter">
     <widget class="QKeySequenceEdit" name="set_rta_hotkey"/>
    </item>
    <item row="15" column="8" colspan="3" alignment="Qt::AlignLeft|Qt::AlignVCenter">
     <widget class="QKeySequenceEdit" name="set_rta_reset_hotkey"/>
    </item>
    <item row="0" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="welcome_text">
      <property name="text">
       <string>Welcome to the Universal In-Game Timer for Minecraft!</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="auto_stop_check">
      <property name="text">
       <string>Automatically stop both timers if End Poem is triggered (1.13+ only)</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="opacity_text">
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="11">
     <widget class="QLineEdit" name="browse_field"/>
    </item>
    <item row="24" column="4" alignment="Qt::AlignRight">
     <widget class="QRadioButton" name="dark_theme_button">
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="browse_leave_empty_text">
      <property name="text">
       <string>(If left empty it'll look for the default Minecraft directory)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="author_text">
      <property name="text">
       <string>By NinjaSnail1080</string>
      </property>
     </widget>
    </item>
    <item row="31" column="0" colspan="12">
     <widget class="QPushButton" name="continue_button">
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="browse_text">
      <property name="text">
       <string>Select your Minecraft directory unless it's already filled in below:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="8" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLabel" name="rta_hotkey_text">
      <property name="text">
       <string>RTA timer start/stop hotkey:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="8" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLabel" name="rta_reset_hotkey_text">
      <property name="text">
       <string>RTA timer reset hotkey:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="rta_timer_check">
      <property name="text">
       <string>RTA Timer</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="12" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QCheckBox" name="auto_start_check">
      <property name="text">
       <string>Automatically start RTA timer on world load (1.7.2+ only)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="8" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLabel" name="rta_interval_text">
      <property name="text">
       <string>RTA update interval (default: 0ms):</string>
      </property>
     </widget>
    </item>
    <item row="10" column="8" colspan="3">
     <widget class="QLineEdit" name="rta_interval_field"/>
    </item>
    <item row="11" column="0" colspan="8" alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLabel" name="igt_interval_text">
      <property name="text">
       <string>IGT update interval (default: 50ms):</string>
      </property>
     </widget>
    </item>
    <item row="11" column="8" colspan="3">
     <widget class="QLineEdit" name="igt_interval_field"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
